.github/workflows/test.yml

```diff
@@ -84,8 +84,10 @@ jobs:
     strategy:
       matrix:
         containerd: [v1.4.5, v1.6.1, main]
+        buildkit-worker: [oci, containerd]
```

Comment:
We seriously have to deflake CI before expanding the test matrix

https://github.com/containerd/nerdctl/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fci

Reply:
Until that we can just test the containerd worker (after BK v0.10.0 GA)

Reply:
Enabled the test only with containerd worker